libraries/botframework-config/tests/loadAndSave.test.js

```diff
@@ -1,212 +1,204 @@
     });
 
-    it("LoadAndSaveBotFileSync", async () => {
-        var secret = bf.BotConfiguration.generateKey();
-        var config = bf.BotConfiguration.loadSync(testBotPath);
+    it('LoadAndSaveBotFileSync', async function () {
+        const secret = bf.BotConfiguration.generateKey();
+        const config = bf.BotConfiguration.loadSync(testBotPath);
 
         // saveAsSync
         config.saveAsSync(saveBotPath, secret);
 
-        var config2 = bf.BotConfiguration.loadSync(saveBotPath, secret);
+        const config2 = bf.BotConfiguration.loadSync(saveBotPath, secret);
         fs.unlinkSync(saveBotPath);
-        assert.deepEqual(config.services, config2.services, "configs should be same");
+        assert.deepEqual(config.services, config2.services, 'configs should be same');
 
         // saveSync
         config2.name = 'saveSync';
         config2.saveSync(secret);
-        var config3 = bf.BotConfiguration.loadSync(saveBotPath, secret);
+        let config3 = bf.BotConfiguration.loadSync(saveBotPath, secret);
         fs.unlinkSync(saveBotPath);
         assert.equal(config3.name, config2.name, "didn't save");
 
         // save
         config2.name = 'save';
         await config2.save(secret);
-        var config3 = bf.BotConfiguration.loadSync(saveBotPath, secret);
+        config3 = bf.BotConfiguration.loadSync(saveBotPath, secret);
         fs.unlinkSync(saveBotPath);
         assert.equal(config3.name, config2.name, "didn't save");
     });
 
-    it("SaveWithNullThros", async () => {
-        var secret = bf.BotConfiguration.generateKey();
-        var config = bf.BotConfiguration.loadSync(testBotPath);
+    it('SaveWithNullThrows', async function () {
+        const secret = bf.BotConfiguration.generateKey();
+        const config = bf.BotConfiguration.loadSync(testBotPath);
 
-        try {
-            config.saveAsSync(null, secret);
-            assert.fail("saveAsSync with null should throw");
-        } catch (err) { }
+        assert.throws(() => config.saveAsSync(null, secret), new Error('missing path'));
 
-        try {
-            await config.saveAs(null, secret);
-            assert.fail("saveAs with null should throw");
-        } catch (err) { }
+        await assert.rejects(config.saveAs(null, secret), new Error('missing path'));
     });
 
-    it("CantLoadWithoutSecret", async () => {
-        let secret = bf.BotConfiguration.generateKey();
-        var config = await bf.BotConfiguration.load(testBotPath);
+    it('CantLoadWithoutSecret', async function () {
+        const secret = bf.BotConfiguration.generateKey();
+        const config = await bf.BotConfiguration.load(testBotPath);
         await config.saveAs(saveBotPath, secret);
 
-        try {
-            await bf.BotConfiguration.load(saveBotPath);
-            assert.fail("Load should have thrown due to no secret");
-        }
-        catch (Error) { }
-    });
-
-    if ("CantSaveWithoutSecret", async () => {
```

Comment:
Note: This was deleted because:

1. This whole package/library is functionally deprecated
2. [It won't throw anyway](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botframework-config/src/botConfiguration.ts#L145
)